src/common/JSHandle.ts

```diff
@@ -957,9 +957,7 @@ export class ElementHandle<
     assert(boundingBox.height !== 0, 'Node has 0 height.');
 
     const layoutMetrics = await this._client.send('Page.getLayoutMetrics');
-    // Fallback to `layoutViewport` in case of using Firefox.
-    const { pageX, pageY } =
-      layoutMetrics.cssLayoutViewport || layoutMetrics.layoutViewport;
+    const { pageX, pageY } = layoutMetrics.layoutViewport;
```

Comment:
As a temp workaround we can get the cssPixel / deprecatedPixel ratio and normalize the layoutViewport.pageX, layoutViewport.pageY by that ratio.

Reply:
smth like 
```
const pixelRatio = layoutMetrics.cssLayoutViewport.clientWidth / layoutMetrics.layoutViewport.clientWidth;
```

Reply:
```suggestion
    const { pageX, pageY } = layoutMetrics.cssVisualViewport;
```